files/generate-vars

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/python
+try:
+    import configparser
+except ImportError:
+    import ConfigParser as configparser
+import os.path
+import shlex
+import subprocess
+import sys
+
+from subprocess import call
+
+
+class GenerateMappingFile():
+    prefix = "[Generate Mapping File] "
+    ansible_play = "../examples/dr_generate_var.yml"
+    dr_tag = "generate_mapping"
+    dr_conf_file = "dr.conf"
+
+    def run(self):
+        print("\n%sStart Generate Variable Mapping File "
+              "For oVirt Ansible Disaster Recovery" % self.prefix)
+        site, username, password, ca, output_file = self._init_vars()
+        print("\n%ssite address: %s \n"
+              "%susername: %s \n"
+              "%spassword: \n"
+              "%sca file location: %s \n"
+              "%soutput file location: %s \n"
+              % (self.prefix,
+                 site,
+                 self.prefix,
+                 username,
+                 self.prefix,
+                 self.prefix,
+                 ca,
+                 self.prefix,
+                 output_file))
+
+        external_vars = "site={} username={} password={} ca={} var_file={}" \
+                        .format(site, username, password, ca, output_file)
+        command = "ansible-playbook {} -t {} -e '{}'" \
+                  .format(self.ansible_play, self.dr_tag, external_vars)
+        # TODO: Remove shell True
+        call(command, shell=True)
+        print("\n%sFinished generating variable mapping file "
+              "for oVirt ansible disaster recovery" % self.prefix)
+        print("\n%sVar file is: '%s'" % (self.prefix, output_file))
```

Comment:
Don't this contain any sensitive information?

Reply:
It contains all the members in the setup.
Basically the person which run this script should also have privilladges to view this file.
I guess that we better be safe then sorry so I will drop this print.

Reply:
Now I noticed that I only mention the location of the mapping file and not its content, so I think that this is relevant enough for the user.
I will change the variable name from output_file to var_file_location to make it more clear